applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -242,14 +240,89 @@ def FillAnalyticSubModelPartsWithNewParticles(self):
         #analytic_particle_ids = [elem.Id for elem in self.spheres_model_part.Elements]
         #self.analytic_model_part.AddElements(analytic_particle_ids)
 
+    def SkinForCube(self):
```

Comment:
Let's remove these two functions added. It seems nobody is calling them, and they should have never been in this file, but in a derived from this.